pkg/build/controller/build/build_controller.go

```diff
@@ -834,4 +837,27 @@ func setBuildCompletionTimestampAndDuration(build *buildapi.Build, podStartTime
 		update.setStartTime(*startTime)
 	}
 	update.setDuration(now.Rfc3339Copy().Time.Sub(startTime.Rfc3339Copy().Time))
+
+	if pod != nil && len(pod.Status.ContainerStatuses) != 0 && pod.Status.ContainerStatuses[0].State.Terminated != nil {
+		msg := pod.Status.ContainerStatuses[0].State.Terminated.Message
+		if len(msg) != 0 {
+			parts := strings.Split(msg, "\n")
+			// 6 will get us 5 lines because the last entry in parts is an empty string.
```

Comment:
well, if the message is \n-terminated, that is - could do strings.Split(strings.TrimRight(msg, "\n"), "\n")